Get Involved

```diff
@@ -0,0 +1,44 @@
+# Get Involved & Build the ODC
+
+Make sure to check out our [CONTRIBUDING docs](https://github.com/OpenDataforWeb3/Resources/blob/main/CONTRIBUTING.md) to learn how you can collaborate with us on Github.
+
+We've started Working Groups for the ODC! Publishing & Comms, Engineering & Infra, Biz Dev
+
+### Publishing & Comms Working Group
+Responsibilities: In charge of creating public facing content and keeping the organizations content up to date in various locations
+
+Twitter Management (Posts, Spaces, etc)
+Discord Management (Announcements, channels, etc)
+Website Content (messaging and publishing)
+Github Content
+Podcast Management
+Dealing with Marketing Vendors
+Managing any other public forums of the ODC
+
```

Comment:
The list above isn't defined and so comes up as a single long sentence. You'll just need to add hyphens